plugins/pulp_rpm/plugins/distributors/iso_rsync_distributor/distributor.py

```diff
@@ -0,0 +1,111 @@
+import logging
+import gettext
+
+from pulp.common.config import read_json_config
+from pulp.plugins.distributor import Distributor
+from pulp.plugins.rsync import configuration
+
+from pulp_rpm.plugins.distributors.iso_rsync_distributor import publish
+from pulp_rpm.common.ids import TYPE_ID_ISO
+
+_ = gettext.gettext
```

Comment:
This should be avoided. Instead import gettest at L#2 as:   `from gettext import gettext as _`.